splinterd/man/splinterd.1.md

```diff
@@ -111,10 +119,10 @@ OPTIONS
   configuration file.
 
 `--config-dir CONFIG-DIR`
-: Specifies the directory containing configuration files.
-  (Default: `/etc/splinter`.)
-
-  This option overrides the `SPLINTER_CONFIG_DIR` environment variable, if set.
+: Specifies the directory containing Splinter configuration files. (Default:
+  `/etc/splinter`, unless `SPLINTER_CONFIG_DIR` or `SPLINTER_HOME` is set.)
+  This option overrides any value set by the environment variables or
```

Comment:
Omit this statement, as it is always true of all CLI options. (Same for instances that follow.)

Reply:
Updated